LondonBritishLibrary/orient/BLorient12859.xml

```diff
@@ -0,0 +1,437 @@
+                                directly above seeking protection against <persName ref="ETH1508Gallai">Gāllā</persName>, 
+                                <persName ref="ETH2043sanqel">Šānqǝllā</persName> and <foreign xml:lang="gez">ʾArami</foreign> (non-believers).</note>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ጸሎት፡ በእንተ፡ ኵሉ፡ ኃይለ፡ ፀላዒ፡ ወፀር፡ ቀጅ፡ መቀጅ፡ እልጓሕስም፡ <gap reason="ellipsis"/>
+                                በኃይለ፡ ዝንቱ፡ አስማቲከ፡ ንፍድ፡ ሙቦል፡ ወምጽልአ፡ አዕይንቲሆሙ፡ ለሰብአ፡ አረሚ፡ ወ<persName ref="ETH2043sanqel">ሻንቅላ፡</persName>
+                                ከመ፡ ኢይንሥኡ፡ ፀብአ፡ ላዕሌየ፡ ለቀቲለ፡ ሰብእናየ፡ ለገብርከ፡ <gap reason="lost"/> ወይኵኑ፡ ከመ፡ ዕብን፡ ፍዙዛነ፡ 
+                                <persName ref="ETH1508Gallai">ጋላ፡</persName> ወ<persName ref="ETH2043sanqel">ሻንቅላ፡</persName> ወአረሚ፡ 
+                                <gap reason="ellipsis"/> ድፍን፡ ልቦሙ፡ ወአጽልም፡ አእይንቲሆሙ፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i12">
+                            <title ref="LIT6477PPAynaTela">Prayer against enemies and for being loved</title>
+                            <note>Written in the right column under the tenth picture and in the left column under the eleventh picture. 
+                                The beginning of the prayer is taken from <ref type="work" corresp="LIT2000Mazmur">Psalm 3</ref>. 
+                                The letter <foreign xml:lang="gez">ዥ</foreign> is represented by the more ancient or irregular form with four little 
+                                strokes marking the sixth order: 
+                                <figure><graphic url="https://betamasaheft.eu/resources/images/snippets/BLorient12996/sign2.png" height="30px">
+                                <desc>Symbol as represented in the catalogue by Stefan Strelcyn</desc></graphic></figure>.</note>
+                            <incipit xml:lang="gez">እግዚኦ፡ ሚበዝኁ፡ እለ፡ ይሣቅዩኒ፡ <gap reason="ellipsis"/> በዥርሮ፡ ድንፍ፡ ልቦሙ፡ ወአጽልም፡ አዕይንቲሆሙ፡ ወአጽርዕ፡
+                                እንግልጋሆሙ፡ ወአብህም፡ አፋሆሙ፡ ወአልስህ፡ ኅምዞሙ፡ ከመ፡ ኢይትናገሩኒ፡ ነገረ፡ ኅሱም፡ ዘእንበለ፡ ፍቅር፡ ወሰላም፡ ለገብርከ፡ <gap reason="lost"/></incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i13">
+                            <title ref="LIT5888AsmatPrayer">Prayer against enemies</title>
+                            <note>Prayer against various enemies fighting the owner of the scroll with all kinds of weapons. Text is written in the first 
+                                or second column under the eleventh picture.</note>
+                            <incipit xml:lang="gez">ለአናንያ፡ ወአዛርያ፡ ወሚሳኤል፡ ስምከ፡ እጼውዕ፡ እንዘ፡ እብል፡ በከመ፡ ጸውዓት፡ እምከ፡ እንዘ፡ ትብል፡ ስምከ፡ ልራፑን፡ ሐራፑን፡ 
+                                ሹሐል፡ <gap reason="ellipsis"/> በኃይለ፡ ዝንቱ፡ አስማቲከ፡ አርኅቆመ፡ እምኔየ፡ ለእለ፡ ይደ፡ ብዩኒ፡ አእላፍ፡ በየማንየ፡ ወአእላፍ፡ በጸጋምየ፡ 
```

Comment:
Here I also think Strelcyn marks that the word divider in the middle of ይደ፡ብዩኒ፡ is `<surplus>`